aws/sagemaker/llm-instance-recommend/lmi_recommender/lmi_config_recommender.py

```diff
@@ -32,6 +32,21 @@
     }
 }
 
+dynamic_batch_deepspeed = {
+    'smoothquant': {'model_category': {'llama-13b', 'llama-7b', 'gptneox', 'bloom', 'gptj'},
+               'instance': ['g4', 'g5', 'p4', 'p5']},
+    'strategy': {0.1: {"batch_size": 4},
+                 0.3: {"batch_size": 8},
+                 0.5: {"batch_size": 32},
+                 0.7: {"batch_size": 64}},
+    'int8': {
+        "engine": "DeepSpeed",
```

Comment:
parallel_loading?

It's pretty important for smoothquant, since you don't want to wait for calibration N times for N workers, and parallel loading should always work when smoothquant works due to mmaploader enhancements